google/gke/gke-how-to/deploy-pro-gke-cluster.rst

```diff
@@ -0,0 +1,137 @@
+
+To create a cluster out of two or more already-running MicroK8s instances, use the
+``microk8s add-node`` command. The MicroK8s instance on which this command is run
+will be the master of the cluster and will host the Kubernetes control plane.
+
+Following the example VMs, run the command in ``k8s-main``.
+
+::
+
+	root@k8s-main:~# microk8s add-node
+	From the node you wish to join to this cluster, run the following:
+	microk8s join 10.128.0.7:25000/3a9547446eae54765ecc0bc522321968/079215076aa1
+
+	Use the '--worker' flag to join a node as a worker not running the control plane, eg:
+	microk8s join 10.128.0.7:25000/3a9547446eae54765ecc0bc522321968/079215076aa1 --worker
+
+	If the node you are adding is not reachable through the default interface you can use one of the following:
+	microk8s join 10.128.0.7:25000/3a9547446eae54765ecc0bc522321968/079215076aa1
+
+Note that, the IP address, port, and token generated here will be project specific.
+You will not have the same values as above. Find more info in the
+`microk8s clustering doc <https://microk8s.io/docs/clustering>`_.
+
+In the worker nodes, execute the respective command to join the cluster as a worker.
+
+::
+
+	root@k8s-worker-1:~# microk8s join 10.128.0.7:25000/3a9547446eae54765ecc0bc522321968/079215076aa1 --worker
+	Contacting cluster at 10.128.0.7
+
+	The node has joined the cluster and will appear in the nodes list in a few seconds.
+
+	This worker node gets automatically configured with the API server endpoints.
+	If the API servers are behind a loadbalancer please set the '--refresh-interval' to '0s' in:
+		/var/snap/microk8s/current/args/apiserver-proxy
+	and replace the API server endpoints with the one provided by the loadbalancer in:
+		/var/snap/microk8s/current/args/traefik/provider.yaml
+
+In the control plane VM, you should be able to see that the nodes have joined the cluster using
+the ``kubetl get nodes`` command.
+
+::
+
+	root@k8s-main:~# microk8s kubectl get nodes --output=wide
+	NAME           STATUS   ROLES    AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION        CONTAINER-RUNTIME
+	k8s-worker-2   Ready    <none>   2m13s   v1.26.1   10.128.0.10   <none>        Ubuntu 20.04.5 LTS   5.4.0-1021-gcp-fips   containerd://1.6.8
+	k8s-main       Ready    <none>   78m     v1.26.1   10.128.0.7    <none>        Ubuntu 20.04.5 LTS   5.4.0-1021-gcp-fips   containerd://1.6.8
+	k8s-worker-1   Ready    <none>   16m     v1.26.1   10.128.0.8    <none>        Ubuntu 20.04.5 LTS   5.4.0-1021-gcp-fips   containerd://1.6.8
+
+You can also check out the cluster-info using the ``kubectl cluster-info`` command.
+
+::
+
+	root@k8s-main:~# microk8s kubectl cluster-info
+	Kubernetes control plane is running at https://127.0.0.1:16443
+
+	To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+
+
+Conclusion
+~~~~~~~~~~
+
+You now have a Ubuntu Pro Kubernetes cluster running on Ubuntu Pro nodes in GCP.
```

Comment:
Can you please add a one-liner to show **how** we know we are running in a Pro cluster? I guess you can use either the `ua` or `pro` CLI clients to assert that. So something like:

```suggestion
You now have an Ubuntu Pro Kubernetes cluster running in GCE. Your Pro subscription can be verified on each of the provisioned nodes by running:

#### TODO: 

And, from the control plane node (``k8s-main`` in the example above), you'll also find the _KUBECONFIG_ files at ``TODO``, which you can then use to control your Pro Kubernetes cluster from any working environment with a Kubernetes client.
```

Reply:
Done.